frame/staking-rewards/src/lib.rs

```diff
@@ -341,9 +374,20 @@ pub mod pallet {
 
 	#[pallet::storage]
 	#[pallet::getter(fn stakes)]
+	// Review [Andy]
+	// Could be beneficial to use a double map here with the prefix being the reward pool
+	// id as we have this relation embedded in the Stake struct and would help in operations when
+	// we wish to remove a pool and some group update on stakes in a pool.
```

Comment:
reward pool id has been removed and replaced with the staked asset id as the key in the map